scripts/reporting/statsGenElastic/package-lock.json

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
we use yarn over npm (yarn is generally faster), not a big deal in general, but might be better to use yarn instead for consistency.